app/models/vmdb_table.rb

```diff
@@ -6,6 +6,35 @@ class VmdbTable < ApplicationRecord
 
   has_one  :latest_hourly_metric,  -> { VmdbMetric.where(:capture_interval_name => 'hourly', :resource_type => 'VmdbTable', :timestamp => VmdbMetric.maximum(:timestamp)) }, :as => :resource, :class_name => 'VmdbMetric'
 
+  # index_name1,unique1,col11,col12|index_name2,unique2,col21,col22
+  virtual_attribute :all_indexes, :string, :arel => (lambda do |t|
+    t.grouping(
+      Arel::Nodes::SqlLiteral.new(
+        "SELECT
+           string_agg(
+           distinct ix.relname || ',' || indisunique || ',' ||
+          regexp_replace(pg_get_indexdef(indexrelid), '^[^\\)]*\\(([^\\)]*)\\).*$', '\\1'), '|')
+        FROM pg_class t
+        INNER JOIN pg_index i ON t.oid = i.indrelid
+        INNER JOIN pg_class ix ON ix.oid = i.indexrelid
+        WHERE t.relname = #{t.name}.name"
+      )
+    )
+    # if we want only the primary index, add "AND i.indisprimary"
+  end)
+
+  # viable for vmdb_table_evm - defined in vmdb_table to make query from base class work
```

Comment:
Can you make this comment more clear?  I don't understand what this is supposed to be telling me.